js/__internal/grids/grid_core/views/m_grid_view.ts

```diff
@@ -599,13 +601,13 @@ const resizingControllerMembers = {
   },
   _checkSize(checkSize) {
```

Comment:
Question.

Why do we need the ```checkSize``` argument here?
As far as I can see, if we pass ```checkSize: false``` the ```_checkSize``` function returns false o_O
Maybe we can don't call the ```_checkSize``` function in these cases at all?